packages/paste-core/components/menu/src/MenuButton.tsx

```diff
@@ -6,9 +6,9 @@ import {Button} from '@twilio-paste/button';
 
 export type MenuButtonProps = MenuPrimitiveButtonProps & ButtonProps;
 
-const MenuButton = React.forwardRef<HTMLButtonElement, MenuButtonProps>((props, ref) => {
+const MenuButton = React.forwardRef<HTMLButtonElement, MenuButtonProps>(({element = 'MENU_BUTTON', ...props}, ref) => {
```

Comment:
Seems to be the only place where you're not using the constant element name. My gut said it felt a little unnecessary to begin with, but it's something I don't really mind about either way.

We should however either; use it everywhere or use it nowhere.